index.html

```diff
@@ -495,6 +495,288 @@ <h2>
+                }
+              ]
+            }
+          },
+          "user_preferences": {
+            "color_scheme": {
+              "dark": {
+                "icons": [
+                  {
+                    "src": "icon-dark.png"
+                  }
+                ]
+              }
+            }
+          }
+        }
+      </pre>
+      <p>
+        In this example, if the user’s primary language is Español, but 
+        their preferred color scheme was set to "dark", the icon supplied 
+        would be the dark version and not the localized one. For this reason,
+        it is imperative that properties taking a [=manifest override object=]
+        as their value consider whether any other properties that also enable
+        overrides should be able to be redefined within them. In the above
+        example, the author’s intent would likely have been better realized
+        if `user_preferences` was put before `translations` and the localized
+        context block redefined the `user_preferences` value for that language:
+      </p>
+      <pre class="json">
+        {
+          "lang": "en-US",
+          "icons": [
+            {
+              "src": "icon.png",
+              "sizes": "128x128",
+              "type": "image/png"
+            }
+          ],
+          "user_preferences": {
+            "color_scheme": {
+              "dark": {
+                "icons": [
+                  {
+                    "src": "icon-dark.png"
+                  }
+                ]
+              }
+            }
+          },
+          "translations": {
+            "es": {
+              "icons": [
+                {
+                  "src": "icon-es.png"
+                }
+              ],
+              "user_preferences": {
+                "color_scheme": {
+                  "dark": {
+                    "icons": [
+                      {
+                        "src": "icon-es-dark.png"
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          },
+          
+        }
+      </pre>
+      <p>
+        User agents MAY ignore any override properties they do not support.
+      </p>
+      <p>
+        To <dfn>apply a manifest override object</dfn>, given [=ordered_map=] 
+        |overrides:json|, [=ordered map=] 
+        |manifest:ordered map|, and [=array=] |allowed_properties:array| run the following:
+        </p>
+        <ol class="algorithm">
+          <li>Let |allowed_overrides| be a new [=ordered map=].
+          </li>
+          <li>[=list/For each=] |member:string| of |overrides|:
+            <ol>
+              <li>If |member| is not in |allowed_properties|, continue.
+              </li>
+              <li>Set |allowed_overrides|[|member|] to |overrides[|member|]|.
+              </li>
+            </ol>
+          </li>
+          <li>Let |overriden_manifest| be the result of creating a new [=ordered map=] 
```

Comment:
@marcoscaceres I’m not sure the best way  to express this, but in JavaScript it would be merging the objects using the spread operator:

```js
let overriden_manifest = {
  …manifest,
  …allowed_overrides
};
```

Reply:
See [map/iterate](https://infra.spec.whatwg.org/#map-iterate) ... something like:

> [=map/For each=] |key| → |value| of |overrides|, set |manifest|[key] to |value|. 

Should do it.

Reply:
I don’t think that handles nested objects as neatly as spread, but I should be able to recursively call it if the value is an object.